src/components/Output/custom-els/TwoUp/index.ts

```diff
@@ -1,14 +1,15 @@
 import * as styles from './styles.css';
 import { PointerTracker, Pointer } from '../../../../lib/PointerTracker';
 
-const legacyClipCompat = 'legacy-clip-compat';
+const legacyClipCompatAttr = 'legacy-clip-compat';
+const verticalAttr = 'vertical';
```

Comment:
Is this meant to be like the disabled attribute, where by presence indicates the state? I was wondering why this wasn't just an `orientation` attribute with a type that has a default of horizontal, but the option of 'vertical'